packages/jest-reporters/src/GitHubActionsReporter.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import stripAnsi = require('strip-ansi');
+import type {AggregatedResult, TestResult} from '@jest/test-result';
+import BaseReporter from './BaseReporter';
+import type {Context} from './types';
+
+const lineAndColumnInStackTrace = /^.*?:([0-9]+):([0-9]+).*$/;
+
+function replaceEntities(s: string): string {
+  const substitutions: Array<[RegExp, string]> = [
+    [/%/g, '%25'],
+    [/\r/g, '%0D'],
+    [/\n/g, '%0A'],
```

Comment:
are there docs for this substitutions? just to leave a code comment pointing to them

Reply:
there is https://github.community/t/set-output-truncates-multiline-strings/16852/3, but I cannot find that in the docs

Reply:
https://github.com/actions/toolkit/blob/b4639928698a6bfe1c4bdae4b2bfdad1cb75016d/packages/core/src/command.ts#L80-L85 is good enough, probably